src/Algebra/Graph/Relation/Symmetric.hs

```diff
@@ -11,27 +11,485 @@
 -----------------------------------------------------------------------------
 module Algebra.Graph.Relation.Symmetric (
     -- * Data structure
-    SymmetricRelation, fromRelation, toRelation,
+    SymmetricRelation(..), fromRelation, toRelation,
```

Comment:
We shouldn't export the constructor of `SymmetricRelation`.

Reply:
I removed the `(..)` now.